src/Stepper/Stepper.js

```diff
@@ -41,7 +41,6 @@ class Stepper extends Component {
 
   static defaultProps = {
     orientation: 'horizontal',
-    activeStep: 0,
```

Comment:
I think this should get in asap. this is a fix, not a breaking change. this means the first step is always active. you can see that on the docs side. I don't think that should be the default behavior, the docs says it isn't so...